do/csi/driver/server_test.go

```diff
@@ -0,0 +1,53 @@
+package driver
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/kubernetes-csi/csi-test/pkg/sanity"
+)
+
+var suite = flag.Bool("suite", false, "run the csi test suite")
+
+func TestDriverSuite(t *testing.T) {
+	if !*suite {
+		t.Skip("skipping test suite. enable by adding the flag '-suite'")
+	}
+
+	socket := "/tmp/csi.sock"
+	endpoint := "unix://" + socket
+	if err := os.Remove(socket); err != nil && !os.IsNotExist(err) {
+		t.Fatalf("failed to remove unix domain socket file %s, error: %s", socket, err)
+	}
+
+	drv := NewDriver(endpoint)
+
+	errCh := make(chan error, 0)
+	go func(drv *Driver) {
+		errCh <- drv.Run()
+	}(drv)
+
+	select {
+	case err := <-errCh:
+		if err != nil {
+			t.Fatal(err)
+		}
+	case <-time.After(time.Second * 2):
```

Comment:
in the future, will we be able to test this without the sleep?

Reply:
The test is running against a blocking server. This is just a leeway so we can capture the error. Unfortunately I can't call `t.Fatal` in a goroutine itself so I only have this technique currently.  Happy to change it to something different.

Reply:
Ok removed it. The test suite has an internal timeout of `one minute` better than nothing.